src/components/sdk/MeetingControls/AudioInputVFControl.tsx

```diff
@@ -0,0 +1,139 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
We no longer include years in our copyright statements.

```
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0
```

Reply:
I tried to remove the year, but got error in the Github workflow
<img width="934" alt="Screen Shot 2021-08-18 at 4 08 15 PM" src="https://user-images.githubusercontent.com/82910472/129983655-27b363d9-8062-4983-a08a-928e9591dec5.png">

Seems like in the React component, the checker still include the year in the copyright. [check-codestyle](https://github.com/aws/amazon-chime-sdk-component-library-react/blob/master/scripts/check-codestyle.js#L149)

Reply:
Will fix it in another PR. Created the [SIM](https://sim.amazon.com/issues/ChimeSDK-1337) to track status.